src/H5Fsuper.c

```diff
@@ -70,6 +70,7 @@ static const unsigned HDF5_superblock_ver_bounds[] = {
     HDF5_SUPERBLOCK_VERSION_2,     /* H5F_LIBVER_V18 */
     HDF5_SUPERBLOCK_VERSION_3,     /* H5F_LIBVER_V110 */
     HDF5_SUPERBLOCK_VERSION_3,     /* H5F_LIBVER_V112 */
+    HDF5_SUPERBLOCK_VERSION_3,     /* H5F_LIBVER_V114 */
```

Comment:
Similar to other comments about versions

Reply:
Fixed